data/mods/ssb/abilities.ts

```diff
@@ -198,6 +198,23 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		},
 	},
 
+	// Isaiah
+	anchorarms: {
+		desc: "Swaps Attack and Defense while Trick Room is up.",
+		name: "Anchor Arms",
+		onUpdate(pokemon) {
+			if (this.field.getPseudoWeather('trickroom')) {
+				if (pokemon.volatiles['powertrick']) return;
+				this.add('-ability', pokemon, 'Anchor Arms');
+				pokemon.addVolatile('powertrick');
+			} else {
+				if (!pokemon.volatiles['powertrick']) return;
+				this.add('-ability', pokemon, 'Anchor Arms');
+				pokemon.removeVolatile('powertrick');
+			}
+		},
+	},
```

Comment:
```suggestion
		onStart(pokemon) {
			if (this.field.getPseudoWeather('trickroom') && !pokemon.volatiles['powertrick']) {
				this.add('-ability', pokemon, 'Anchor Arms');
				pokemon.addVolatile('powertrick');
			}
		},
		onAnyPseudoWeatherChange(pokemon, source, pseudoWeather) {
			if (this.field.getPseudoWeather('trickroom')) {
				if (pokemon.volatiles['powertrick']) return;
				this.add('-ability', pokemon, 'Anchor Arms');
				pokemon.addVolatile('powertrick');
			} else {
				if (!pokemon.volatiles['powertrick']) return;
				this.add('-ability', pokemon, 'Anchor Arms');
				pokemon.removeVolatile('powertrick');
			}
		},
	},
```

Reply:
`pokemon` should be `this.effectState.target`

Reply:
ur just mad that im prettier than u